dbms/src/AggregateFunctions/AggregateFunctionGroupBitmap.h

```diff
@@ -53,4 +54,105 @@ class AggregateFunctionBitmap final : public IAggregateFunctionDataHelper<Data,
 };
 
 
+template <typename T, typename Data, typename Policy>
+class AggregateFunctionBitmapL2 final : public IAggregateFunctionDataHelper<Data, AggregateFunctionBitmapL2<T, Data, Policy>>
+{
+public:
+    AggregateFunctionBitmapL2(const DataTypePtr & type)
+            : IAggregateFunctionDataHelper<Data, AggregateFunctionBitmapL2<T, Data, Policy>>({type}, {}){}
+
+    String getName() const override { return Data::name(); }
+
+    DataTypePtr getReturnType() const override
+    {
+        return std::make_shared<DataTypeNumber<T>>();
+    }
+
+    void add(AggregateDataPtr place, const IColumn ** columns, size_t row_num, Arena *) const override
+    {
+        Data & data_lhs = this->data(place);
+        const Data & data_rhs = this->data(static_cast<const ColumnAggregateFunction &>(*columns[0]).getData()[row_num]);
```

Comment:
Better to use `assert_cast` here instead of `static_cast`, like in other aggregate functions.

Reply:
@akuzm I don't see how use-after-free happen at dbms/src/AggregateFunctions/AggregateFunctionGroupBitmap.h:74. Could you help on this?

Reply:
@akuzm @abyss7 After replacing `static_cast` with `assert_cast`, `SELECT groupBitmapOr(31)` still crashes the server(segfault at `AggregateFunctionGroupBitmap.h:74`), but not the debug server(built with `cmake -DCMAKE_BUILD_TYPE=Debug ..`, cli gets error `Code: 368. DB::Exception: Received from localhost:9000. DB::Exception: Bad cast from type DB::ColumnVector<unsigned char> to DB::ColumnAggregateFunction`. ). 
Do you think this change is good enough?

Reply:
The error message means that you are casting the column to a wrong type: `ColumnVector<usigned char>` to `ColumnAggregateFunction`. The next step would be to correct this wrong cast.

Reply:
@akuzm It's hard for `createAggregateFunctionBitmapL2` to distinguish groupBitmapOrMerge(AggregateFunction(groupBitmap, UInt32)) from groupBitmapOr(UInt32).
So I decide to remove support of groupBitmapOrMerge(AggregateFunction(groupBitmap, UInt32)).
Please review #7128.